### DISCUSSION TITLE
Issues and PRs

### DISCUSSION THREAD

**Discussion by abdullahceylan** [quran/quran.com-frontend-next#718] [General] [answered]
Salaam all,

Firstly, JazakAllah Khair for being a great example to the Muslim community by creating this great project and making it open source.

I was exploring the issue lists yesterday to contribute and help you and I noticed that there is no description, screenshots, targets, etc. info in the issues. I also didn't see any descriptions in PR details. I don't think this is good for the newcomers who would like to pick some issues for helping and also for future references.

The devs who pick and send the fix are probably here since the beginning and they acknowledge what is going on but it's difficult for others to involve and contribute. At least, for me. I also have noticed that none of the issues have `label`. Maybe you can add `good for first issue` label for the basic ones and it can help newcomers to do something for good.

@tifa2UP @osamasayed

**Comment by tifa2UP** [comment, marked as answer]
Wa alaikum alsalam @abdullahceylan, thank you for your feedback. You do make good points. 

I will work with the team to label some issues as `good for first issue` and add proper documentation to them insha'Allah.

**Reply by abdullahceylan** [reply]
JazakAllah Khair for answering and taking an action for this.

**Comment by Samir-Mulla** [comment]
> Salaam all,
> 
> Firstly, JazakAllah Khair for being a great example to the Muslim community by creating this great project and making it open source.
> 
> I was exploring the issue lists yesterday to contribute and help you and I noticed that there is no description, screenshots, targets, etc. info in the issues. I also didn't see any descriptions in PR details. I don't think this is good for the newcomers who would like to pick some issues for helping and also for future references.
> 
> The devs who pick and send the fix are probably here since the beginning and they acknowledge what is going on but it's difficult for others to involve and contribute. At least, for me. I also have noticed that none of the issues have `label`. Maybe you can add `good for first issue` label for the basic ones and it can help newcomers to do something for good.
> 
> @tifa2UP @osamasayed 


Assalamualikum warahmatullahi wabarakaatuh 
my name is Samir Mulla , i am from india  my problem is i am not able to download the audio file of surah on quran.com platform please quickly solve this bug... 
insha allah you and your team will solve this bug.......
